module/evdi_connector.c

```diff
@@ -85,7 +85,7 @@ static struct drm_encoder *evdi_best_single_encoder(struct drm_connector
 	int enc_id = connector->encoder_ids[0];
 
 	return drm_encoder_find(connector->dev,
-#if KERNEL_VERSION(4, 16, 0) <= LINUX_VERSION_CODE
+#if KERNEL_VERSION(4, 15, 0) < LINUX_VERSION_CODE
```

Comment:
it should stay `<=` though or else it will skip version 4.15.0 specifically (and work from 4.15.1)